bgpd/bgp_evpn.c

```diff
@@ -260,9 +260,6 @@ static struct irt_node *import_rt_new(struct bgp *bgp,
 {
 	struct irt_node *irt;
 
-	if (!bgp)
```

Comment:
If we drop this then:

```c
	if (!irt) {
		irt = import_rt_new(bgp, &eval_tmp);
		assert(irt);
	}
```
seems not necessary too?

Reply:
Done, thanks.